src/reducers/RootReducer.js

```diff
@@ -9,8 +9,6 @@ import bales from './BalesReducer';
 import createBaleModal from './CreateBaleModalReducer';
 import editBaleModal from './EditBaleModalReducer';
 import createPocketModal from './CreatePocketModalReducer';
-import date from './GatherReducer';
-import hour from './GatherReducer';
 import travelImage from './GatherReducer';
```

Comment:
RootReducer is used to combine differents reducers, not to combine attributes of a particular reducer